libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4312,6 +4313,17 @@ void GCS_MAVLINK::send_sys_status()
         errors4); // errors4
 }
 
+
```

Comment:
some extra lines we can probably remove here.

Reply:
Fixed.